### PR TITLE
Update Schema+Creator.swift

### PR DESCRIPTION
**Pull request by chriscombs**
Added optional name field to override the parent format as mentioned in #161

### CODE REVIEW

**Comment by chriscombs** [comment]
I guess the `entity` parameter becomes redundant with this, but it was the least disruptive method of doing it.

File: Sources/Fluent/Schema/Schema+Creator.swift [hunk 1]
```diff
@@ -128,12 +128,13 @@ extension Schema {
...
         public func parent<E: Entity>(
             _ entity: E.Type = E.self,
+            name: String? = nil,
             optional: Bool = false,
             unique: Bool = false,
             default: NodeRepresentable? = nil
         ) {
             fields += Field(
-                name: "\(entity.name)_id",
+                name: name ?? "\(entity.name)_id",
```
Comment (LoganWright) [inline Sources/Fluent/Schema/Schema+Creator.swift]: should this also be `\(name)_id` or should the external label perhaps be `customId: String?` or perhaps a secondary function of some sort. Just thinking out loud.

File: Sources/Fluent/Schema/Schema+Creator.swift [hunk 2]
```diff
@@ -125,21 +125,33 @@ extension Schema {
...
             default: NodeRepresentable? = nil
+            ) {
+            parent(customId: "\(entity.name)_id", 
+                   optional: optional, 
+                   unique: unique, 
+                   default: `default`)
+        }
+
+        public func parent(
+            customId: String,
```
Comment (tannernelson) [inline Sources/Fluent/Schema/Schema+Creator.swift]: `idKey` might be a better name for this. `customId` seems like its referring to the id values.

Comment (chriscombs) [inline Sources/Fluent/Schema/Schema+Creator.swift]: Sounds good, I'll update that

**Pull request merged**